components/logins/src/db.rs

```diff
@@ -1232,7 +1211,51 @@ mod tests {
             ..Login::default()
         });
 
-        db.delete(login.guid_str()).unwrap();
         assert!(&unique_login_check.is_ok())
     }
+
+    #[test]
+    fn test_delete() {
```

Comment:
Thanks for adding these, can you check that the tombstones are still present after deletion too? E.g. that there's at a record with that id which was deleted in the local table with is_deleted == true.

I'm fine if you just run a sql query for this directly in the test code, I don't think an api for this is exposed and there's no need to overthink it. Ping me if you need help

Reply:
@thomcc Will do! Thanks for keeping me honest. 😬